### PR TITLE
Additional interpolation implementations

### PR DESCRIPTION
**Pull request by motin**
Hi Boris,
We found everpolate to be the best basis for interpolation features in https://github.com/Gapminder/gsheets-gm-functions, thanks for putting this out here! 
In the process of implementing the interpolation features, the improvements summarized in this PR were made. Let me know what you think :)

### CODE REVIEW

**Comment by BorisChumichev** [comment]
Hello @motin,
Sorry for late rsponse. PR looks great. I have one question though. Can you please provide a use case example for `fillMissing` method?

File: lib/fillMissing.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+var help = require('./help')
+
+module.exports = fillMissing
+
+/**
+ * Evaluates interpolating step function at the set of numbers
```
Comment (BorisChumichev) [inline lib/fillMissing.js]: This comment line looks like an irrelevant one :)

**Comment by motin** [comment]
> I have one question though. Can you please provide a use case example for fillMissing method?

Sure: Olympic medals for countries. Such datasets usually only include the countries that actually got medals. A country that has a datapoint for 3 medals year 2005 and 5 medals year 2007 should be assumed to have 0 medals the years for which datapoints are missing.

**Comment by BorisChumichev** [comment]
Thank you, @motin . This makes sense. But I think the method naming can be improved though. I feel like filling missing points is what interpolation is all about in general thus having a method with name `fillMissing` is a bit odd. The method name should represent the way it fills the missing points: "step", "linear", "exponential" etc. In this case we are filling missing points with a single value i.e constant. So the method name could be `constant` or `value`. I would go with `value`. What do you think?

**Comment by motin** [comment]
> filling missing points is what interpolation is all about in general 

Sooo true :) - renamed to value.

**Comment by BorisChumichev** [comment]
@motin I think it's ready to be merged :)
Now it's time to add documentation on this and publish a new version on npm. This is what I will take care of on this weekends. Thanks for your conribution 👍
